### PR TITLE
TILA-2379: Refund webhook endpoint

### PR DESCRIPTION
**Pull request by saaste**
## Change log
- Added missing configs to `.env.example`
- Added a new endpoint that handles refund webhooks from Verkkokauppa

## Notes
Our model is still missing a field that is required a bit later. I left a TODO comment for this. There is already task for it.

## Deployment reminder
- No changes required

### CODE REVIEW

**Review by nicobav — CHANGES_REQUESTED** [review]

File: api/webhook_api/views.py [hunk 1]
```diff
@@ -212,3 +213,74 @@ def create(self, request):
...
+            "timestamp",
+        ]
+        for field in required_field:
+            if field not in request.data:
+                raise WebhookError(
+                    message=f"Required field missing: {field}", status_code=400
+                )
+
+        namespace = request.data.get("namespace", None)
+        type = request.data.get("type", None)
```
[the remark this answers is not in the document]
Comment (saaste) [reply api/webhook_api/views.py]: Renamed to `webhook_type` 👍

File: api/webhook_api/views.py [hunk 2]
```diff
@@ -212,3 +213,74 @@ def create(self, request):
...
+            },
+        ),
+        responses=default_responses,
+    )
+    def create(self, request):
+        try:
+            self.validate_request(request)
+
+            remote_id = request.data.get("orderId", "")
+            # refund_id = request.data.get("refundId", "")
```
[the remark this answers is not in the document]
Comment (saaste) [reply api/webhook_api/views.py]: Cleaned 🧹

**Comment by saaste** [comment]
Checks fails because of outdated Django with CVE. #773 has the update so I'll retry the checks once that is merged.

**Review by nicobav — APPROVED** [review]
👍

**Pull request merged**